framework/contrib/ice_updater/include/LibcurlUtils.h

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
```

Comment:
How about the idea of putting all of the ICE agnostic stuff in `framework/include/utils/url` or something?  It basically is MOOSE code... there's no real reason to keep it separated over in the contrib area.  It wasn't an external project that we brought in or anything...